README.md

```diff
@@ -16,3 +17,61 @@ To install dependent modules, double click on ```install_dependencies.py```. Alt
 Please note that you might have to change the LABRADHOST environmental variable after installation. For instructions on how to do this, look up on CampbellGroup wiki.
 
 To install the ok module, you need to go the software folder on the Campbell server and install the corresponding package.
+
+# Conventions
+New code should...  
+* Be Python 2 & 3 compatible  
```

Comment:
I think this is going to be hard to implement because nobody really uses python 3. Maybe more specific, for example, 

Use 
```python
print(str)
``` 
instead of 
```python
print str
``` 
and 
```python
except Exception as e:
```
instead of 
```python
except Exception, e:
```